core/src/test/java/io/questdb/test/jit/CompiledFilterIRSerializerTest.java

```diff
@@ -784,6 +784,12 @@ public void testVarcharNullConstant() throws Exception {
         assertIR("(varchar_header avarchar)(i64 4L)(<>)(ret)");
     }
 
+    @Test
+    public void testIn() throws Exception {
```

Comment:
Would be great to have some variations in the test, different number of input values, more complex filters, separate tests for error cases like `anint in ()`.